.github/workflows/forge-gcp.yaml

```diff
@@ -71,6 +74,7 @@ jobs:
       - name: Write summary
         run: |
           IMAGE_TAG=${{ steps.determine-test-image-tag.outputs.IMAGE_TAG }}
+          # IMAGE_TAG="devnet"
```

Comment:
might be useful to leave a comment here if we aren't going to delete this

Reply:
Removed.